kubelet/conf.yaml.example

```diff
@@ -13,11 +13,18 @@ instances:
     ### This is the default setting. See next section for legacy clusters.
     ###
     #
-    # url of the kubelet metrics endpoint
-    # Pass an empty string, or set the cadvisor_port option to disable
-    # prometheus metrics collection
+    # DEPRECATED: please specify `cadvisor_metrics_endpoint` instead
     # metrics_endpoint: http://10.8.0.1:10255/metrics/cadvisor
     #
+    # url of the cadvisor metrics prometheus endpoint
+    # Pass an empty string, or set the cadvisor_port option to disable
+    # cadvisor metrics collection
+    # cadvisor_metrics_endpoint: http://10.8.0.1:10255/metrics/cadvisor
+    #
+    # url of the kubelet metrics prometheus endpoint
+    # Pass an empty string to disable kubelet metrics collection
+    # kubelet_metrics_endpoint: http://10.8.0.1:10255/metrics
+    #
```

Comment:
This ended up being much more straightforward to be able to turn off kublet metrics and/or cadvisor metrics by setting the option to `""` than trying to share `kubelet_api_endpoint` between them.

Reply:
yes, a deprecation warning would be nice.